haddock-api/haddock-api.cabal

```diff
@@ -189,6 +189,7 @@ test-suite spec
                , containers
                , deepseq
                , directory
+               , exceptions
```

Comment:
The previous lack of CI shows! 😄

Reply:
You don't say…  
I'm really playing cat and mouse here :P